templates/docs/examples/patterns/strips/dark.html

```diff
@@ -0,0 +1,15 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Strips / Dark{% endblock %}
+
+{% block standalone_css %}patterns_strip{% endblock %}
+
+{% set is_paper = true %}
```

Comment:
Should this be set to `is_dark` instead?

Reply:
Technically yes, but in case of strips I want to showcase that setting `is-dark` on the strip itself changes it's background.

If `is-dark` is set on body strip would be transparent.